HW03/Igor_Hrybinyk/lib/mentor.rb

```diff
@@ -0,0 +1,35 @@
+class Mentor
+  include StudentObserver
+  attr_reader :mentor_name
```

Comment:
```suggestion
  attr_reader :name
```
It seems to me better to use such a variable name

Reply:
done